lib/oli/delivery/sections.ex

```diff
@@ -2586,6 +2590,61 @@ defmodule Oli.Delivery.Sections do
     end)
   end
 
+  defp build_list_children_activity_ids([]), do: []
+
+  defp build_list_children_activity_ids(children_activities) do
+    tuple_of_children_activities =
+      children_activities
+      |> process_activity_ids()
+
+    Enum.concat(Tuple.to_list(tuple_of_children_activities))
```

Comment:
Nit: good chance to use the pipe operator here :)

Reply:
nice suggestion